std-bits/table/src/main/java/org/enso/table/data/column/operation/map/numeric/DoubleLongBooleanOpWithSpecialNumericHandling.java

```diff
@@ -0,0 +1,63 @@
+package org.enso.table.data.column.operation.map.numeric;
+
+import org.enso.table.data.column.operation.map.TernaryMapOperation;
+import org.enso.table.data.column.operation.map.MapOperationProblemBuilder;
+import org.enso.table.data.column.storage.numeric.DoubleStorage;
+import org.enso.table.data.column.storage.Storage;
+import org.enso.table.error.UnexpectedTypeException;
+import org.graalvm.polyglot.Context;
+
+import java.util.BitSet;
+
+/** An operation expecting a numeric argument and returning a number. */
+public abstract class DoubleLongBooleanOpWithSpecialNumericHandling extends TernaryMapOperation<Double, DoubleStorage> {
```

Comment:
It feels to me like class `DoubleLongBooleanOpWithSpecialNumericHandling` is **super** specific. Will we get more inheritors of it other than round?

I would personally prefer to just create a `DoubleRoundOp` class. I like abstraction but it feels like it's unnecessary here.

Reply:
So, same class, but different name?

Reply:
Mostly.

What I mean I would not make it an `abstract` class anymore but just an implementation of the Round operation.

Especially as you will notice, if you want to return Integer column for `decimal_places <= 0` case, you will need more special handling of the return type yet. So IMO it's good to just create a standalone class specialized for rounding. Most of the code from this class can be retained indeed.

Reply:
Done.